doc/nrf/config_and_build/bootloaders/qspi_xip_split_image.rst

```diff
@@ -0,0 +1,295 @@
+.. _qspi_xip_split_image:
+
+Using QSPI XIP split image
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The QSPI XIP split image feature lets you gain more flash storage space for applications by splitting application code into two parts:
+
+* Code that runs on the internal flash.
+* Code that runs on a supported external flash over Quad Serial Peripheral Interface (QSPI) using Execute in place (XIP).
```

Comment:
```suggestion
* Code that runs on supported external flash memory over the Quad Serial Peripheral Interface (QSPI) using Execute in Place (XIP).
```

Reply:
I was leaning towards your edit, but then I saw it is spelled in sentence style in the PS https://docs.nordicsemi.com/bundle/ps_nrf52840/page/qspi.html#ariaid-title6

Reply:
Fixed